TU20Bot/Configuration/Config.cs

```diff
@@ -30,7 +30,46 @@ public class Config {
             "Greetings",
             "Howdy"
         };
-        
+
+        public readonly string[,] origNames = {
+            {"Dev","Narula"},
+            {"Timur","Khayrullin"},
+            {"Zaynah","nolastname"},
+            {"Yifan","Wang"},
+            {"Tahmeed","Naser"},
+            {"Shrena","Sribalan"},
+            {"Rick","(O20)"},
+            {"Pranav","Vyas"},
+            {"Karen","Truong"},
+            {"Muhammad Muizz","nolastname"},
+            {"Vince","Li"},
+            {"Alex","Li"},
+            {"nofirstname","Nolan"},
+            {"Borna","Sadeghi"},
+            {"Hargun","nolastname"},
+            {"Markos","Georghiades"},
+            {"Muhammad Muizz","Zafar"},
+            {"Taylor desgroup","Whatley"},
+            {"Viktor fasest","Korolyuk"},
+            {"Denys","Linkov"},
+            {"Muhammad Ali","Syed"},
+        };
+
+        public List<string> emails = new List<string>{
+            "johndoe@tu20.com",
+            "johndoe@yahoo.com",
+            "johndoe@outlook.com",
+            "johndoe@gmail.com",
+            "johndoe@hubspot.com",
+            "johndoe@pepipost.com",
+            "johndoe@zohomail.com",
+            "johndoe@aolmail.com",
+            "johndoe@pleasekillme.com",
+            "johndoe@nothingmail.com",
+        };
+
+        public static Dictionary<ulong, string> userEmailId = new Dictionary<ulong, string>();
```

Comment:
But if they're static, every time I instantiate the `Config` class by using `new Config()`, the data stored in the `userEmailId` is lost, since it's initialized again :(

Do you think it would be a good idea to instantiate the Config class only once throughout the whole program (at the start of the program, in the `Program.cs` file), `userEmailId` will retain its data? 🤔 That way I won't have to make it static.

Reply:
You should never instantiate Config. :/

We *have* on instance shared across the program. This is explained in other messages. If you can't find the instance in a certain place you can ask me.